client/state/happychat/middleware.js

```diff
@@ -18,6 +25,16 @@ export const requestTranscript = ( connection, { getState, dispatch } ) => {
 	);
 };
 
+export const sendEventMessage = ( connection, { getState }, action ) =>{
+	const state = getState();
+	const currentUser = getCurrentUser( state );
+	if ( isHappychatClientConnected( state ) &&
+		isHappychatAcceptingChats( state ) &&
```

Comment:
My understanding (and the tests I've done) is that `happychat.isAvailable` is only updated when a client is [first accepted](https://github.com/Automattic/wp-calypso/blob/master/client/state/happychat/actions.js#L125).